src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-size: center;
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+
+  &:nth-child(-n + 4) {
```

Comment:
Using ':nth-child' with a negative value is not standard for BEM methodology. It's better to add modifier classes to the stars based on their active state, which can be applied directly in the HTML.